package/gluon-l3roamd/files/etc/modules.d/gluon-l3roamd

```diff
@@ -0,0 +1 @@
+cfg80211
```

Comment:
allright then, I will remove this line from my image an report back. I know I added when I was already well into the project but cannot remember the initial motivation.